cocos/dragon-bones/ArmatureDisplay.ts

```diff
@@ -426,13 +428,31 @@ export class ArmatureDisplay extends UIRenderer {
             this._frameCache.enableCacheAttachedInfo();
         }
     }
-
+    /**
+     * @en
+     * The socket nodes contains the attached node be registered Synchronous motion
+     * transformation with bones.
+     * @zh
+     * Socket nodes存储挂载到骨骼上的节点，可以随骨骼做同步运动变换。
+     */
     get socketNodes () { return this._socketNodes; }
```

Comment:
And i think should add `readonly` for return type. Of course, can modify in next version.